### PR TITLE
New package: novaboot, perl-Expect (dependency of novaboot)

### PR DESCRIPTION
**Pull request by ametisf**
I haven't figured out how to reopen the last pull request, so here is another one with hopefuly everything fixed now :)

### CODE REVIEW

File: srcpkgs/novaboot/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+depends="perl perl-Expect"
+short_desc="Tool that automates booting of operating systems on hardware or in qemu"
+maintainer="adam <ametisf@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/wentasah/novaboot"
+distfiles="https://github.com/wentasah/novaboot/archive/${version}.tar.gz"
+checksum=f1a8781d9bb470d631014063986fb89334747bb718e3acef33d24a35f4359548
+
+do_build() {
+	make
```
Comment (maxice8) [inline srcpkgs/novaboot/template]: this can be replaced with the gnu-makefile build_style

Comment (ametisf) [reply srcpkgs/novaboot/template]: Fixed. Thank you.

File: srcpkgs/novaboot/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+do_install() {
+	make PREFIX=/usr DESTDIR=${DESTDIR} install
```
[the remark this answers is not in the document]
Comment (ametisf) [reply srcpkgs/novaboot/template]: Fixed.

**Comment by Hoshpak** [comment]
Please change the commit messages to include the version number, e.g. `New package: novaboot-20180323`. The templates look good to me now.

**Pull request merged**